frontend/docsettings.lua

```diff
@@ -244,9 +279,15 @@ function DocSettings:flush(data)
                 ffiutil.fsyncDirectory(sidecar_file)
             end
 
+            -- move cover file to the metadata file location
+            if self.cover_file and util.splitFilePathName(self.cover_file) ~= sidecar_dir then
+                local mv_bin = Device:isAndroid() and "/system/bin/mv" or "/bin/mv"
+                ffiutil.execute(mv_bin, self.cover_file, sidecar_dir)
+            end
+
             self:purge(sidecar_file) -- remove old candidates and empty sidecar folders
 
-            break
+            return sidecar_dir
```

Comment:
Even if we do not use `sidecar_file` anywhere?

Reply:
If I would have added that myself, yes, even if we don't use it yet anywhere, and I just need the second return value - just because it would look less random. And I would have added a comment `return sidecar_file, sidecar_dir -- callers might have some use for that`.

But I'm ok if you don't, just because it will avoid having to use `-- luacheck: no unused` :)